Install LIBMINCConfig.cmake in a directory where CMake can find it

--- libminc-libminc-2-3-00.old/CMakeLists.txt	2015-09-11 21:04:28.000000000 +0200
+++ libminc-libminc-2-3-00/CMakeLists.txt	2017-09-05 16:24:18.286228198 +0200
@@ -419,7 +419,7 @@
   ${CMAKE_CURRENT_BINARY_DIR}/UseLIBMINC.cmake @ONLY)
 
 # config for install dir
-SET(LIBMINC_USE_FILE_CONFIG    "${CMAKE_INSTALL_PREFIX}/${LIBMINC_INSTALL_LIB_DIR}/Use${LIBMINC_EXTERNAL_LIB_PREFIX}LIBMINC.cmake")
+SET(LIBMINC_USE_FILE_CONFIG    "${CMAKE_INSTALL_PREFIX}/${LIBMINC_INSTALL_LIB_DIR}/cmake/${PROJECT_NAME}/Use${LIBMINC_EXTERNAL_LIB_PREFIX}LIBMINC.cmake")
 SET(LIBMINC_INCLUDE_DIRS_CONFIG ${CMAKE_INSTALL_PREFIX}/${LIBMINC_INSTALL_INCLUDE_DIR} )
 SET(LIBMINC_LIBRARY_DIRS_CONFIG ${CMAKE_INSTALL_PREFIX}/${LIBMINC_INSTALL_LIB_DIR})
 SET(LIBMINC_STATIC_LIBRARIES_CONFIG   "")
@@ -452,7 +452,7 @@
      ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/Use${LIBMINC_EXTERNAL_LIB_PREFIX}LIBMINC.cmake 
      ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/${LIBMINC_EXTERNAL_LIB_PREFIX}LIBMINCConfig.cmake
     DESTINATION 
-     ${LIBMINC_INSTALL_LIB_DIR}
+     ${LIBMINC_INSTALL_LIB_DIR}/cmake/${PROJECT_NAME}
     COMPONENT Development)
   
 ENDIF(LIBMINC_INSTALL_LIB_DIR AND NOT LIBMINC_INSTALL_NO_DEVELOPMENT)
